includes/HelpCenterFeatureHooks.php

```diff
@@ -0,0 +1,69 @@
+<?php
+namespace NewfoldLabs\WP\Module\HelpCenter;
+
+use NewfoldLabs\WP\Module\Data\SiteCapabilities;
+
+use function NewfoldLabs\WP\Module\Features\disable as disableFeature;
```

Comment:
The `disableFeature` doesn't seem to be used anywhere .. is this for a future enhancement?